core/isolate.rs

```diff
@@ -204,6 +209,13 @@ impl Isolate {
     self.dyn_import = Some(Arc::new(f));
   }
 
+  pub fn set_js_error_create<F>(&mut self, f: F)
```

Comment:
Please add a triple slash comment documenting the default behavior and pupose of js_error_create